transport/http/handler_test.go

```diff
@@ -48,12 +48,20 @@ func TestHandlerSucces(t *testing.T) {
 	headers.Set(TTLMSHeader, "1000")
 	headers.Set(ProcedureHeader, "nyuck")
 	headers.Set(ServiceHeader, "curly")
+	headers.Set(BaggageHeaderPrefix+"Foo", "bar")
+	headers.Set(BaggageHeaderPrefix+"BAR", "baz")
 
 	rpcHandler := transporttest.NewMockHandler(mockCtrl)
 	httpHandler := handler{rpcHandler}
 
 	rpcHandler.EXPECT().Handle(
-		transporttest.NewContextMatcher(t, transporttest.ContextTTL(time.Second)),
+		transporttest.NewContextMatcher(t,
+			transporttest.ContextTTL(time.Second),
+			transporttest.ContextBaggage(transport.Headers{
```

Comment:
optional idea, what is `transporttest.ContextBaggage` was a custom type alias for `transport.Headers`, but it also implements the option interface, so this would look like:
```go
transporttest.NewContextMatcher(t,
  transporttest.ContextTTL(time.Second),
  transporttest.ContextBaggage{
    "foo": "bar",
    "bar": "baz",
  },
  [...]
```

(actually the same thing could be done for `ContextTTL` but the usage doesn't change)